test/parallel/test-buffer-alloc.js

```diff
@@ -464,6 +464,16 @@ assert.strictEqual(
 // Regression test for https://github.com/nodejs/node/issues/3496.
 assert.strictEqual(Buffer.from('=bad'.repeat(1e4), 'base64').length, 0);
 
+// Regression test for https://github.com/nodejs/node/issues/11987.
+assert.strictEqual(Buffer.from('w0  ', 'base64').length, 1);
+assert.deepStrictEqual(Buffer.from('w0  ', 'base64'),
+                       Buffer.from('w0', 'base64'));
+assert.deepStrictEqual(Buffer.from('w0  ', 'base64'), Buffer.from([195]));
+assert.strictEqual(
+  Buffer.from('3v6YpUFyK0/hitA2tCDIfdYKw0 g ', 'base64').toString('base64'),
+  '3v6YpUFyK0/hitA2tCDIfdYKw0g='
+);
+
```

Comment:
@addaleax I think adding all these tests is redundant. This one conveys intent the best:

```js
assert.deepStrictEqual(Buffer.from('w0  ', 'base64'),
                       Buffer.from('w0', 'base64'));
```

If this test passes but one of the other proposed tests fails, that means base64 encoding is broken in some other way, which would cause one of the many existing tests to fail.